tasks/config.ts

```diff
@@ -79,15 +80,14 @@ export function load() {
       };
     });
 
-    const networkName = hre.network.name.toUpperCase();
     const chainId = +(await hre.getChainId());
 
     const blockchainConfig = {
       ["BDX_ADDRESS"]: (await getBdx(hre)).address,
       ["NATIVE_TOKEN_WRAPPER_ADDRESS"]: (await getWeth(hre)).address,
       ["EXTERNAL_USD_STABLE"]: EXTERNAL_USD_STABLE[hre.network.name],
       ["SECONDARY_EXTERNAL_USD_STABLE"]: SECONDARY_EXTERNAL_USD_STABLE[hre.network.name],
-      ["SOVRYN_SWAP_NETWORK_ADDRESS"]: chainId === chainIds.rsk ? chainSpecificComponents[networkName].sovrynNetwork : undefined,
+      ["SOVRYN_SWAP_NETWORK_ADDRESS"]: chainId === chainIds.rsk ? chainSpecificComponents[hre.network.name].sovrynNetwork : undefined,
```

Comment:
General thought.
I used the chain name in this repo for the settings, but on the FE & BR I used the chainId.

Maybe we should switch and use it here as well?
The reason I used the name is because rthe chain id is a promise unlike the name..

Even if we're keeping it, you can drop the compare part

```suggestion
      ["SOVRYN_SWAP_NETWORK_ADDRESS"]: chainSpecificComponents[hre.network.name].sovrynNetwork,
```

Reply:
if we'll change it to the chainId than the config itself should change and the key should be the id instead of the name - I find it better with the name as it's easier to read and you don't need to remember all the chain ids.

I prefer to keep the comparison as it is more explicit this way that the `sovrynNetwork` should exist only on RSK. otherwise, someone could think it's a missing config on other networks